sendgrid/helpers/permissions/permissions.py

```diff
@@ -0,0 +1,305 @@
+                if (".read" in permission):
+                    read_only_permissions.append(permission)
+            return read_only_permissions
+        else:
+            print(
+                "There is no permissions group with that given name {0}".format(group))
+            return None
+
+    '''
+        Extra features implemented to add/remove permissions from each permissions list and group.
+        This is done so we don't have to update permisisons when we want to remove or add one permission.
+    '''
+    
+    #   Add permission to permissions list
+    #   inputs:
+    #       list_name:string variable containing name of list
+    #       permission:string variable containing permission to be added
+    #   outputs:
+    #       success:boolean True if operation is successful, False if not
+    def add_permission_to_list(self, list_name, permission):
+        if (list_name in self.available_api_key_permissions):
+            if (not permission in self.api_key_permissions[list_name]):
+                self.api_key_permissions[list_name].append(permission)
+                return True
+            else:
+                print("Permission {0} is already inside of list {1}.".format(
+                    list_name, permission))
+                return False
+        else:
+            print("Error: Invalid list name.")
+            return False
+
+
+    #   Add permission to permissions group
+    #   inputs:
+    #       group_name:string variable containing name of group
+    #       permission:string variable containing permission to be added
+    #   outputs:
+    #       success:boolean True if operation is successful, False if not
+    def add_permission_to_group(self, group_name, permission):
+        if (group_name in self.available_common_api_key_groups):
+            if (not permission in self.common_api_key_groups[group_name]):
+                self.common_api_key_groups[group_name].append(permission)
+                return True
+            else:
+                print("Permission {0} is already inside of group {1}.".format(
+                    group_name, permission))
+                return False
+        else:
+            print("Error: Invalid group name.")
+            return False
+
+
+    #   Delete permission from permissions list
+    #   inputs:
+    #       list_name:string variable containing the name of list
+    #       permission:string variable containing the permissions to be deleted
+    #   outputs:
+    #       success:boolean True if operation is successful, False if not
+    def delete_permission_from_list(self, list_name, permission):
+        if (list_name in self.available_api_key_permissions):
+            if (permission in self.api_key_permissions[list_name]):
+                self.api_key_permissions[list_name].remove(permission)
+                return True
+            else:
+                print("Permission {0} is not inside group {1}.".format(
```

Comment:
When you try to remove a value from a list that it doesn't have, you get a `ValueError`:
```
>>> l = ['foo']
>>> l.remove('bar')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: list.remove(x): x not in list
```
Maybe you could do something similar here if the user gives an incorrect permission or list name, and let the user handle the exception.